app/models/podcast_episode.rb

```diff
@@ -75,6 +75,10 @@ def user_name
     podcast_title
   end
 
+  def podcast_image
+    ProfileImage.new(podcast).get(width: 90)
+  end
```

Comment:
as this is only called in the serializer, would it make sense to keep it there in a block?